### PR TITLE
Update double-conversion recipe

### PR DESCRIPTION
**Pull request by Vrondir**

### CODE REVIEW

File: dev-libs/double-conversion/double_conversion-3.1.5.recipe [hunk 1]
```diff
@@ -63,5 +67,6 @@ INSTALL()
 
 TEST()
 {
-	make check
+	make
```
Comment (Begasus) [inline dev-libs/double-conversion/double_conversion-3.1.5.recipe]: No need for "make" here, you can remove the line, make process has already been done in BUILD()

File: dev-libs/double-conversion/double_conversion-3.1.5.recipe [hunk 2]
```diff
@@ -45,6 +48,7 @@ BUILD()
 		-DINSTALL_CMAKE_DIR=$libDir/cmake -DINSTALL_BIN_DIR=$binDir \
 		-DINSTALL_LIB_DIR=$libDir -DINSTALL_INCLUDE_DIR=$includeDir \
 		-DBUILD_SHARED_LIBS=ON
+	cmake . -DBUILD_TESTING=ON
```
Comment (Begasus) [inline dev-libs/double-conversion/double_conversion-3.1.5.recipe]: Keep this line in par with the previous one, don't call cmake twice:
```diff
-		-DBUILD_SHARED_LIBS=ON
-	cmake . -DBUILD_TESTING=ON
+		-DBUILD_SHARED_LIBS=ON -DBUILD_TESTING=ON
```

**Comment by Begasus** [comment]
LGTM

**Comment by korli** [comment]
SONAME change. qt5 might need a rebuild.

**Comment by Begasus** [comment]
Checking a rebuild for qt5 atm, but indeed needs a revbump
qT5

**Comment by extrowerk** [comment]
Maybe you could clean the recipe a bit up:

CMake Warning:
  Manually-specified variables were not used by the project:

    INSTALL_BIN_DIR
    INSTALL_CMAKE_DIR
    INSTALL_INCLUDE_DIR
    INSTALL_LIB_DIR

**Comment by Begasus** [comment]
> SONAME change. qt5 might need a rebuild.

PR for qt5 made :)

**Pull request merged**